java-native/src/main/java/com/baeldung/jvm/JVMBitVersion.java

```diff
@@ -0,0 +1,28 @@
+package com.baeldung.jvm;
```

Comment:
Let's be a bit more specific (no pun intended) with the package name, e.g. com.baeldung.jvmbitversion